src/Raven.Client/Http/ReadBehavior.cs

```diff
@@ -2,9 +2,9 @@
 {
     public enum ReadBehavior
     {
-        LeaderOnly,
-        LeaderWithFailover,
-        LeaderWithFailoverWhenRequestTimeSlaThresholdIsReached,
+        ConversationNodeOnly,
```

Comment:
I think I just transcribed it from our discussion about this. I will change this to "CurrentNodeOnly" kind of naming